Resources/Prototypes/_CD/Entities/Mobs/Player/ipc.yml

```diff
@@ -0,0 +1,7 @@
+- type: entity
```

Comment:
Is there any reason to separate this into an abstract and non-abstract part? No other species do this from the looks of things.

Reply:
dunno, avali do it